test/ILLink.RoslynAnalyzer.Tests/RequiresUnreferencedCodeAnalyzerTests.cs

```diff
@@ -4,61 +4,15 @@
 
 using System;
 using System.Threading.Tasks;
+using ILLink.CodeFix;
 using Microsoft.CodeAnalysis.Testing;
 using Microsoft.CodeAnalysis.Text;
 using Xunit;
-using VerifyCS = ILLink.RoslynAnalyzer.Tests.CSharpCodeFixVerifier<
```

Comment:
Same here, I think you can revert most, if not all changes.

Reply:
I still see value in using a single generic VerifyDiagnostic that is shared between the analyzers, any reason to revert the changes?

Reply:
Commented offline: there's a lot of benefit in sharing, but I think we should introduce a new base class for the tests, which is a bigger change. We could do it in this PR, but probably easier to do it in a new PR and get this one merged.